### ISSUE TITLE
Hide / Compact UI at runtime

### ISSUE THREAD

**Issue by imsnif** [zellij-org/zellij#1400]
Plugins / Layouts in general and the Zellij UI specifically should support a compact mode and a "hidden ui" mode.

The compact UI should include one bar with a list of open tabs and the current mode.

Depends on: #742

**Comment by a-kenji** [comment]
This seems to be a duplicate of #694.

**Comment by imsnif** [comment]
True. It was a little hard for me to find the other one because its title talks about the implementation rather than the user feature. This one is in the roadmap SVG though (which is not super trivial to change) - is it ok if we leave it open and close both once this is implemented?

**Comment by aral** [comment]
This would also be very helpful when doing screencasts/live streams. I’d love to be able to hide the interface after performing an action so I have as much screen space as available given how small the available space is when you’re presenting on 1080p with large fonts, etc. :)

**Comment by imsnif** [comment]
Hey @aral - while I very much agree, I just want to make sure you know about `zellij --layout compact`? You aren't yet able to switch to it at runtime, but if you're already comfortable with the keybindings and don't need the hints, it gives you all that you need in just 1 line.

**Comment by aschleifer** [comment]
While I also like the idea of being able to switch, these days I rarely start zellij in the default layout, since I'm quiet familiar with the key bindings now.

But still especially for new people it could be a helpful feature.

**Comment by artfulrobot** [comment]
Not sure if this is the right place, but in compact mode, when a pane is full-screened, I frequently find myself forgetting if I have other panes (on this tab) or not. i.e. it would be good to have something showing the number of panes in the tab, or an F or something to indicate I'm in full screen mode. The presence of floating windows could also be useful.

e.g. current:
```
Zellij (main)  NORMAL » *Tab #1 » Tab #2 »                     » HORIZONTAL »
```

idea: `3+2` might mean 3 panes +2 floating windows:

```
Zellij (main)  NORMAL » Tab #1 » Tab #2 »                     » HORIZONTAL 3+2»
```

- a Tab with a single pane would say nothing extra
- a Tab with a single pane + 1 float would have `+1`
- a Tab with 2 panes and no floats would have just `2`

**Comment by imsnif** [comment]
Hey @artfulrobot - all of this is possible to achieve with plugins.
